easybuild/easyconfigs/m/Molpro/Molpro-mpp-2010.1.23-Linux_x86_64.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Molpro'
+easyblock = 'EB_Molpro'
+versionprefix = 'mpp-'
```

Comment:
I fail to see how this is overwritten?

> On 23/09/2015, at 5:54 am, Kenneth Hoste <notifications@github.com> wrote:
> 
> In easybuild/easyconfigs/m/Molpro/Molpro-mpp-2010.1.23-Linux_x86_64.eb:
> 
> > @@ -0,0 +1,23 @@
> > +name = 'Molpro'
> > +easyblock = 'EB_Molpro'
> > +versionprefix = 'mpp-'
> 
> remove, since it's overwritten below?
> 
> —
> Reply to this email directly or view it on GitHub.
>